script/generalhw_scripts/flash_usb_otg.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
as these scripts are only used by tests themselves better move them to data/

Reply:
Fixed.